builtin/k8s/platform.go

```diff
@@ -429,7 +429,9 @@ func (p *Platform) Deploy(
 				}
 
 				if cs.State.Waiting != nil {
-					if cs.State.Waiting.Reason == "ImagePullBackOff" {
+					// TODO: handle other pod failures here
+					if cs.State.Waiting.Reason == "ImagePullBackOff" ||
+						cs.State.Waiting.Reason == "ErrImagePull" {
```

Comment:
Didn't the old version warn in the same way but actually catch more conditions? I'm confused.

Reply:
The old version only looked at `"ImagePullBackOff"`, which happens after `"ErrImagePull"`. I'm not sure if they are exclusively similar, but to me the error we're actually trying to handle here is when it fails to pull the image. I'm not sure what you mean by catching more conditions on the old version, unless there's a change prior to this PR you're talking about :smile:

Reply:
oh I can't read. Nevermind!